tests/test_user_agent.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+import re
+import percy
+from percy.user_agent import UserAgent
+
+class TestPercyUserAgent(unittest.TestCase):
+    def setUp(self):
+        percy_client = percy.Client(config=percy.Config())
+        self.user_agent = UserAgent(percy_client)
+
+    def test_string(self):
+        self.assertTrue(
+            re.match(
+                'Percy/v1 python-percy-client/[.\d]+ \(python/[.\d]+; buildkite\)',
```

Comment:
That was the case @timhaines! Fixed with: e511c93942bfcdc528108cd8337da93b669ea779